src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  text-align: center;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+
+    margin: 0 auto 40px;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__item-name {
+    width: 166px;
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    line-height: 18px;
+    text-align: start;
+    font-weight: 500;
+    color: $main-text-color;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    text-align: start;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text-color;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 23px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-text-color;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    margin-bottom: 16px;
+
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary-text-color;
+  }
+
+  &__price-tag {
+    margin-bottom: 16px;
+
+    font-size: 16px;
+    font-weight: 700;
+    text-align: end;
+    color: $main-text-color;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-size: 14px;
+    height: 40px;
+    font-weight: 700;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $button-text-color;
+
+    background: $button-bg-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    &:hover {
+      color: $button-bg-color;
+      background: white;
+      border-color: $button-bg-color;
```

Comment:
It works only for some small project which won't need a lot of support further. What if we gonna need the same color in some other place, not for a button? Will this name be a good choice?